templates/dashboard/contact_detail.html

```diff
@@ -70,4 +72,37 @@ <h1 class="h3">
   </div>
 {% endblock %}
 
+{% block script %}
+    {% if current_user.is_premium() %}
+    <script>
+    const MAX_BYTES = 2048; // 2KiB
+
+    function drop(event) {
```

Comment:
As this drag and drop feature is used in both for mailbox and contact pages, I wonder if we can put it into a common file, for example we can put the code into `static/js/utils/drag_on_drop.js` and use a more generic class name (e.g. `.drag-drop-area` instead of `#pgp-public-key`), what do you think?

Reply:
I moved all the js into a file that contains a function to init the drag n' drop. The id of the text area can be passed as a param so we don't tie ourselves to having a magic name. Each page can have it's own and even if we have two of them in one page we can reuse the same with two ids.